external/storm-mqtt/core/src/main/java/org/apache/storm/mqtt/common/MqttUtils.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.storm.mqtt.common;
+
+
+import org.fusesource.mqtt.client.QoS;
+
+public class MqttUtils {
+
+    private MqttUtils(){}
+
+    public static QoS qosFromInt(int i){
+        QoS qos = null;
+        switch(i) {
+            case 0:
+                qos = QoS.AT_MOST_ONCE;
+                break;
+            case 1:
+                qos = QoS.AT_LEAST_ONCE;
+                break;
+            case 2:
+                qos = QoS.EXACTLY_ONCE;
```

Comment:
Why are we taking int and converting into Qos enum instance? Can not we use Qos enum directly in Options instead of int? enum is serializable, so it can be passed as argument to spout and bolt constructors.